content/post/12-zappos-offers-new-hires/index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Zappos Offers New Hires $3,000 to Quit After 4 Weeks"
+date: 2010-11-16T00:00:00-07:00
+draft: false
+categories:
+- Entrepreneurship
+- Zen
+
+---
+
+This article was posted two years ago, but it’s still a pretty cool concept.  Basically, Zappos (the online shoe site that Amazon recently bought) will train new hires for 4 weeks, and then offers them $3,000 to quit.  Check it out for yourself:
+http://www.businessweek.com/smallbiz/content/sep2008/sb20080916_288698.htm
```

Comment:
This won't automatically render as a URL by the markdown renderer. Replace with:

```
[Check it out for yourself](http://www.businessweek.com/smallbiz/content/sep2008/sb20080916_288698.htm)
```